.github/workflows/publish-website.yml

```diff
@@ -11,43 +11,43 @@ jobs:
   publish-website:
     runs-on: ubuntu-20.04
     steps:
-    - uses: actions/checkout@v2.4.0
-    - name: Prepare for publication to GitHub Packages
-      uses: actions/setup-node@v2.5.1
-      with:
-        node-version: '16.x'
-        registry-url: 'https://registry.npmjs.org'
-    - name: Build API docs
-      run: |
-        npm ci
-        npm run build-api-docs
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
-      with:
-        python-version: 3.8
-    - name: Install Python dependencies
-      run: |
-        python -m pip install --upgrade pip
-        cd docs/api/
-        pip install -r requirements.txt
-    - name: Build website
-      run: |
-        npm run build-docs-preview-site
-        touch docs/dist/.nojekyll
-    - name: Deploy to GitHub Pages
-      run: |
-        git config user.name $GITHUB_ACTOR
-        git config user.email $GITHUB_ACTOR@users.noreply.github.com
-        git remote add gh-pages-remote https://x-access-token:$GH_TOKEN@github.com/$GITHUB_REPOSITORY.git
-        git fetch --no-recurse-submodules
-        cd docs
-        git worktree add ./gh-pages gh-pages
-        cd gh-pages
-        git rm -r .
-        cp -r ../dist/. .
-        git add .
-        git commit --message="Deploying to GitHub Pages from $GITHUB_SHA"
-        git push gh-pages-remote gh-pages:gh-pages
-      env:
-        GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
-        GITHUB_REPOSITORY: ${{ github.repository }}
+      - uses: actions/checkout@v2.4.0
+      - name: Prepare for publication to GitHub Packages
+        uses: actions/setup-node@v2.5.1
+        with:
+          node-version: "16.x"
+          registry-url: "https://registry.npmjs.org"
+      - name: Build API docs
+        run: |
+          npm ci
+          npm run build-api-docs
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install Python dependencies
+        run: |
+          python -m pip install --upgrade pip
+          cd docs/api/
+          pip install -r requirements.txt
+      - name: Build website
+        run: |
+          npm run build-docs-preview-site
+          touch docs/dist/.nojekyll
+      - name: Deploy to GitHub Pages
+        run: |
+          git config user.name $GITHUB_ACTOR
+          git config user.email $GITHUB_ACTOR@users.noreply.github.com
+          git remote add gh-pages-remote https://x-access-token:$GH_TOKEN@github.com/$GITHUB_REPOSITORY.git
+          git fetch --no-recurse-submodules
+          cd docs
+          git worktree add ./gh-pages gh-pages
+          cd gh-pages
+          git rm -r .
+          cp -r ../dist/. .
+          git add .
+          git commit --message="Deploying to GitHub Pages from $GITHUB_SHA"
+          git push gh-pages-remote gh-pages:gh-pages
```

Comment:
It may be an idea to move the larger script blocks like this to scripts within the workflows folder, such that they're not just inline in yaml.

Reply:
Fixed in 44c5daf